flybywiresim/a32nx
# chore: Update stable to 0.6.2-rc

Fixes #5442

## Summary of Changes

* fix(lights): switch compatibility

* fix: formatting

* fix: Console/Floor LT switch

* fix: workaround for "NOT IN DATABASE"

Co-authored-by: Saschl <sascharudolf46@gmail.com>

* feat: port updated touchdown fx

* fix: update simvar for ECAM message

* fix: logo LT update code

* fix: separate dome light emission

* fix: formatting

* fix: master warning and master caut pb

* fix: consistent ids in template

* fix: ann lt test for master caut and warn + prevent gear up on ground

* fix: add gear lever wwise event

* fix: formatting
* 
Co-authored-by: Saschl <sascharudolf46@gmail.com>
Co-authored-by: Taz5150 <70781153+Taz5150@users.noreply.github.com>


## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


1. Check that Master WARN/Caution can be pushed and work as expected
2. Check that landing lights and all light switches work, and that lights retract/extend as expected.
3. Check that flight plan is able to be loaded correctly (Simbrief or menu).
4. Check that every button in the cockpit has tooltips, can be interacted (with correct behaviour), every light knob/switch within the cockpit (pedestal lights, etc) all work. Exterior lights have some known issues, and that unfortunately will have to be mitigated in the future 0.7.0 release. 


Known Issue (Won't Fix): 
- Strobe Lights are not fully disabled when off, unusual/unexpected strobe behavior. 
- A/T is connected during TOGA

Fixing:
- Master Caution unable to be cleared
- Missing Tooltips (vs Development)


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
